front/app/components/Form/Components/Controls/MultiSelectControl.tsx

```diff
@@ -47,19 +47,22 @@ const MultiSelectControl = ({
         subtextSupportsHtml
       />
       <Box display="flex" flexDirection="row" overflow="visible">
-        <StyledMultipleSelect
-          value={data}
-          options={options}
-          onChange={(vals) => {
-            setDidBlur(true);
-            handleChange(
-              path,
-              vals.map((val) => val.value)
-            );
-          }}
-          inputId={sanitizeForClassname(id)}
-          disabled={uischema?.options?.readonly}
-        />
+        <Box flexGrow={1}>
```

Comment:
From what I can see this control is only used for registration fields (idea form + survey uses a different multiselect control). I confirmed that the problem was fixed on both the profile page and when signing up in the registration modal.